### ISSUE TITLE
Site deployed guide and API should respect and use stable version tag, not master branch

### ISSUE THREAD

**Issue by resurtm** [yiisoft/yii2#13259]
Hello!

http://www.yiiframework.com/doc-2.0/guide-index.html
http://www.yiiframework.com/doc-2.0/index.html

It turns out the guide and API both shows what is in master branch now. Which is not very good, and can confuse framework users. Developers will have non-standard behavior, and this is quite weird when documentation is for master, but 99% of people use stable (2.0.10 at the moment).

Example:

http://www.yiiframework.com/doc-2.0/guide-input-validation.html#creating-validators
```php
/**
 * @param string $attribute the attribute currently being validated
 * @param mixed $params the value of the "params" given in the rule
 * @param \yii\validators\InlineValidator related InlineValidator instance
 */
function ($attribute, $params, $validator)
```

Third parameter is true for 2.0.11: https://github.com/yiisoft/yii2/blob/master/framework/validators/InlineValidator.php#L72

But not for 2.0.10: https://github.com/yiisoft/yii2/blob/2.0.10/framework/validators/InlineValidator.php#L69

I didn't manage to find similar issue, so please close this if there is something similar.

Cheers,
@resurtm

**Comment by cebe** [comment]
> It turns out the guide and API both shows what is in master branch now. Which is not very good, and can confuse framework users. Developers will have non-standard behavior, and this is quite weird when documentation is for master, but 99% of people use stable (2.0.10 at the moment).

this is going to be fixed on the new website. There will be a switch to choose between 2.0.x 2.1.x and dev-master docs.

We also usually add something like "This parameter is available since version 2.0.11" so that problem is not that big. have been overseen in this case. will add it.

**Comment by cebe** [comment]
fixed, thanks for reporting!
